Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginManager.swift

```diff
@@ -30,66 +30,87 @@ public final class SimplifiedLoginManager {
                 withMFA: MFAType? = nil,
                 loginHint: String? = nil,
                 extraScopeValues: Set<String> = [],
                 completion: @escaping LoginResultHandler) {
+        
+        let fetcher = SimplifiedLoginFetcher(client: client)
+        self.init(client: client, withMFA: withMFA, loginHint: loginHint, extraScopeValues: extraScopeValues, completion: completion, fetcher: fetcher)
+    }
+    
+    @available(iOS, obsoleted: 13, message: "This function should not be used in iOS version 13 and above")
+    init(client: Client,
+                     withMFA: MFAType? = nil,
+                     loginHint: String? = nil,
+                     extraScopeValues: Set<String> = [],
+                     completion: @escaping LoginResultHandler,
+                     fetcher: SimplifiedLoginFetching) {
         self.client = client
         self.withMFA = withMFA
         self.loginHint = loginHint
         self.extraScopeValues = extraScopeValues
         self.completion = completion
+        self.fetcher = SimplifiedLoginFetcher(client: client)
     }
     
     @available(iOS 13.0, *)
-    public init(client: Client,
+    convenience public init(client: Client,
                 contextProvider: ASWebAuthenticationPresentationContextProviding,
                 env: ClientConfiguration.Environment, // TODO: Currently used to decide language.
                 withMFA: MFAType? = nil,
                 loginHint: String? = nil,
                 extraScopeValues: Set<String> = [],
                 withSSO: Bool = true,
                 completion: @escaping LoginResultHandler) {
+        let fetcher = SimplifiedLoginFetcher(client: client)
+        self.init(client: client, contextProvider: contextProvider, env: env, withMFA: withMFA, loginHint: loginHint, extraScopeValues: extraScopeValues, withSSO: withSSO, completion: completion, fetcher: fetcher)
+    }
+    
+    @available(iOS 13.0, *)
+    init(client: Client,
+                contextProvider: ASWebAuthenticationPresentationContextProviding,
+                env: ClientConfiguration.Environment, // TODO: Currently used to decide language.
+                withMFA: MFAType? = nil,
+                loginHint: String? = nil,
+                extraScopeValues: Set<String> = [],
+                withSSO: Bool = true,
+                completion: @escaping LoginResultHandler,
+                fetcher: SimplifiedLoginFetching) {
         self.client = client
         self.withMFA = withMFA
         self.loginHint = loginHint
         self.extraScopeValues = extraScopeValues
         self.completion = completion
         self.withSSO = withSSO
         self._contextProvider = contextProvider
-
+        self.fetcher = fetcher
     }
 }
 
 extension SimplifiedLoginManager {
+    
     /**
-     Prepere and configure Simplified Login View Controller which should be shown modaly
+     Prepare and configure Simplified Login View Controller which should be shown modally
```

Comment:
ah old info - now it is shown automatically ;)

Reply:
Updated it!

Reply:
"which should is shown modally" will be better because we are taking care of showing it

Reply:
I see you changed it! Nice! :)